GDM.HW4.OOP.Classes.Task2/Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GDM.HW4.OOP.Classes.Task2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Worker anton = new Worker();
+            Worker bogdan = new Worker();
+            Worker clark = new Worker();
+            string agregat = "Agregat";
+            int completeAgrigat = 50;
+
+            for (int i = 0; agregat.Length < completeAgrigat; i++)
+            {
+                agregat = anton.JuniorWorker(agregat);
```

Comment:
Hardcoded solution. You shouldn't call specific method. You should call "DoWork" and every worker should do job according to his skills.

Reply:
Additionaly create class Agregat